config/environments/test.rb

```diff
@@ -57,4 +57,10 @@
 
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
+
+  # Enable Faraday request header logging
```

Comment:
See previous comments.

We can remove this one.

Reply:
Also done.